tests/integration/tests/autocomplete.js

```diff
@@ -250,7 +250,7 @@ test('Search Query', async () => {
   const searchValue = await getInputValue(page, '#search');
 
   // search input is filled with query
-  expect(searchValue).toEqual(searchQuery);
+  expect(searchValue).toEqual('test result 1');
```

Comment:
As discussed, this behavior looks good to me at this stage. The comment related to this check should be updated though, to clarify the intention. The query does not persist in the search input anymore.